test/performance/perf-kinesis/src/mz.rs

```diff
@@ -12,6 +12,8 @@ use std::time::Duration;
 use anyhow::Context;
 use tokio_postgres::Client;
 
+use test_util::mz as mz_util;
```

Comment:
The fact that you rename this module everywhere suggests that it has the wrong name. Perhaps `mz_client`, and then you can use that everywhere?